File: Week 2/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n > 8 || n < 1);

    for (int i = 1; i <= n; i++)
    {
        for (int j = 0; j < (n - i); j++)
        {
            printf(" ");
        }
        for (int h = 0; h < i; h++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: Week 6/pset6/mario/mario.py
from cs50 import get_int

n = get_int("Height: ")
while n > 8 or n < 1 or not isinstance(n, int):
    print("Sorry, invalid.")
    n = get_int("Height: ")

for i in range(n):
    for j in range(n-i-1):
        print(" ", end="")
    for h in range(i+1):
        print("#", end="")
    print("")

